core/src/main/scala/org/apache/spark/status/api/v1/api.scala

```diff
@@ -71,7 +71,9 @@ class ExecutorStageSummary private[spark](
     val shuffleWriteRecords : Long,
     val memoryBytesSpilled : Long,
     val diskBytesSpilled : Long,
-    val isBlacklistedForStage: Boolean)
+    val isBlacklistedForStage: Boolean,
+    var executorLogs: Map[String, String],
```

Comment:
I don't like these being var and modified after the fact.  I think we should either include them from the original call or make another call, rather then add them in like this.

Reply:
Done